excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AuthorisationCaseEventParser.java

```diff
@@ -32,52 +32,55 @@
         validateCaseEvents(definitionSheets, definitionSheet, caseTypeReference);
 
         final List<DefinitionDataItem> dataItems = dataItemMap.get(caseTypeReference);
+        final Map<String, List<DefinitionDataItem>> collect = dataItems == null
+            ? null
+            : dataItems.stream().collect(groupingBy(d -> d.getString(ColumnName.CASE_EVENT_ID)));
 
-        if (null == dataItems) {
-            LOG.warn("No data is found for case type '{} in AuthorisationCaseEvents tab", caseTypeReference);
-        } else {
-            LOG.debug("Parsing user roles for case type {}: {} AuthorisationCaseEvents detected", caseTypeReference, dataItems.size());
+        for (EventEntity event : events) {
+            final List<EventACLEntity> parseResults = Lists.newArrayList();
 
-            final Map<String, List<DefinitionDataItem>> collect = dataItems.stream().collect(groupingBy(d -> d.getString(ColumnName.CASE_EVENT_ID)));
+            final String eventReference = event.getReference();
+            LOG.debug("Parsing AuthorisationCaseEvent for case type {}, event {}...", caseTypeReference, eventReference);
 
-            if (null == collect.get(eventReference)) {
-                LOG.warn("No row is defined for case type '{}', event '{}'", caseTypeReference, eventReference);
-                // and let validation handles this Exception
+            if (null == dataItems) {
+                LOG.warn("No data is found for case type '{} in AuthorisationCaseEvents tab", caseTypeReference);
             } else {
-                for (DefinitionDataItem definition : collect.get(eventReference)) {
-                    EventACLEntity entity = new EventACLEntity();
-
-                    parseUserRole(entity, definition, parseContext);
-                    parseCrud(entity, definition);
-                    parseResults.add(entity);
-                    entityToDefinitionDataItemRegistry.addDefinitionDataItemForEntity(entity, definition);
-
-                    LOG.info("Parsing user role for case type '{}', event '{}', user role '{}', crud '{}': OK",
-                        caseTypeReference,
-                        eventReference,
-                        definition.getString(ColumnName.USER_ROLE),
-                        definition.getString(ColumnName.CRUD));
+                LOG.debug("Parsing user roles for case type {}: {} AuthorisationCaseEvents detected", caseTypeReference, dataItems.size());
+
+                if (null == collect.get(eventReference)) {
+                    LOG.warn("No row is defined for case type '{}', event '{}'", caseTypeReference, eventReference);
+                    // and let validation handles this Exception
+                } else {
+                    for (DefinitionDataItem definition : collect.get(eventReference)) {
+                        EventACLEntity entity = new EventACLEntity();
+
+                        parseUserRole(entity, definition, parseContext);
```

Comment:
Unfortunately the logging is different with different fields getting logged.

Parsing of ACL entities follows the same logic across both events and authorisation case field, it would be good to combine those whole methods but I wanted to limit the scope of this performance fix.